block-storage-attacher/tests/e2e/scripts/common.sh

```diff
@@ -491,3 +513,41 @@ function install_blockvolume_plugin {
 	set -e
 	echo "Checking storage plugin and driver status and wait till running"
 }
+function install_portworx_plugin {
+	if [ -z $PORTWORX_HELM_CHART ]; then
+        echo "port worx helm chart not found. Hence exiting"
+        exit 1
+    fi
+    echo "Installing helm chart portworx plugin  .."
+	# INSTALL HELM TILLER (Attempt again, if already installed)
+	echo "Initialize tiller AND Wait till running"
+	helm init --force-upgrade
+	check_pod_state "tiller-deploy"
+	
+        # INSTALL/UPGRADE HELM CHART
+	helm_values_override="--set etcdEndPoint=$ETCD_SET1;$ETCD_SET2,clusterName=$(uuidgen),etcd.credentials=$ETCDCREDS,usedrivesAndPartitions=true,usefileSystemDrive=true"
```

Comment:
You need to also set the `drives` parameter of values.yaml with device path (example `/dev/dm-1`) found in the `kubectl describe pv` output of block attacher PV object. You can see it in annotations and fetch from there.

Reply:
No action taken on this comment. Please check and revert.

Reply:
We need not to provide the drives parameter as port worx has to identify the formatted drives